MainRunner.java

```diff
@@ -0,0 +1,127 @@
+package pack.homework;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class MainRunner {
+    public static void main(String[] args) {
+
+        Scanner reader = new Scanner(System.in);
+
+        double radius = 0;
+        double sideOne = 0;
+        double sideTwo = 0;
+        double sideA = 0;
+        double sideB = 0;
+        double sideC = 0;
+
+        Rectangular myRectangle = new Rectangular(sideOne, sideTwo);
+        Triangle myTriangle = new Triangle(sideA, sideB, sideC);
+        Circle myCircle = new Circle(radius);
+
+        int answer = 0;
+        char choice = 0;
+        int calc = 0;
+
+        ArrayList<Shape> shapes;
+
+        do {
+            do {
+                System.out.println("How many calculations you would like?");
+                calc = reader.nextInt();
+
+                if (calc <= 0) {
+                    System.out.println("Number of calculations must be 1 or higher.");
+                    System.out.println("Try again? y/n");
+                    choice = reader.next().toLowerCase().charAt(0);
+                } else break;
+            } while (choice == 'y');
+
+            shapes = new ArrayList<>(calc);
+
+            System.out.println("MENU | Choose shape:");
+            String[] menuOptions = {
+                "Rectangle area and perimeter",
+                "Circle area and circumference",
+                "Triangle area and perimeter",
+            };
+
+            menuFromArray(menuOptions);
+
+            answer = reader.nextInt();
+
+            if (answer == 1) {
+                int count = 0;
+                for (int i = 0; i < calc; i++) {
+                    System.out.println("For your " + ++count + "." + " rectangle.");
+                    System.out.println("Enter first side of rectangle:");
+                    sideOne = reader.nextDouble();
+                    System.out.println("Enter second side of rectangle:");
+                    sideTwo = reader.nextDouble();
+                    shapes.add(new Rectangular(sideOne, sideTwo));
+                }
+                printResult(shapes);
```

Comment:
You shouldn't calculate all shapes for each entry user makes. You should have a logic which terminates user inputs when he entered number of shapes that he wanted to calculate and then run execution. 
From the performance standpoint, if calculation is more complex and time extensive, running it every time user enters new shape would be time consuming.